src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/WorkflowManager.kt

```diff
@@ -18,14 +19,37 @@ class WorkflowManager(sync: InstanceSync) : InstanceService(sync) {
         aem.prop.boolean("instance.workflowManager.restoreIntended")?.let { set(it) }
     }
 
+    fun schedule(modelId: String, node: Node) = workflow(modelId).schedule(node)
+
     fun workflow(id: String) = Workflow(this, id)
 
+    fun workflow(id: String, options: Workflow.() -> Unit) = Workflow(this, id).run(options)
+
     fun workflows(type: String) = workflows(listOf(type))
 
     fun workflows(types: Iterable<String>) = WorkflowType.ids(types).map { Workflow(this, it) }
 
     fun workflows(vararg types: String) = workflows(types.asIterable())
 
+    fun queryNodes(path: String, type: String): Sequence<Node> = instance.sync.repository {
+        query {
+            path(path)
+            type(type)
+        }.nodeSequence()
+    }.also {
+        if (it.count() == 0) {
+            logger.lifecycle("No resources found on $instance")
```

Comment:
imagine that you are seeing that msg... what does it mean?
what resources is the first question. please add context of input args of that method like

```kotlin
logger.lifecycle("No resources of type '$type' found under path '$path' on $instance")
```

Reply:
don't try to log random messages, please implement them carefully

Reply:
btw `log.info` ; it's a implementation detail, it should be optionally visible